references/cli/addon.go

```diff
@@ -193,16 +191,10 @@ Enable addon for specific clusters, (local means control plane):
 
 // AdditionalEndpointPrinter will print endpoints
 func AdditionalEndpointPrinter(ctx context.Context, c common.Args, k8sClient client.Client, name string) {
-	endpoints, _ := GetServiceEndpoints(ctx, k8sClient, pkgaddon.Convert2AppName(name), types.DefaultKubeVelaNS, c, Filter{})
-	if len(endpoints) > 0 {
-		table := tablewriter.NewWriter(os.Stdout)
-		table.SetColWidth(100)
-		table.SetHeader([]string{"Cluster", "Component", "Ref(Kind/Namespace/Name)", "Endpoint"})
-		for _, endpoint := range endpoints {
-			table.Append([]string{endpoint.Cluster, endpoint.Component, fmt.Sprintf("%s/%s/%s", endpoint.Ref.Kind, endpoint.Ref.Namespace, endpoint.Ref.Name), endpoint.String()})
-		}
-		fmt.Printf("Please access the %s from the following endpoints:\n", name)
```

Comment:
we can keep this line in the function `printAppEndpoints`

Reply:
‘fmt.Printf("Please access the %s from the following endpoints:\n", name)’ this line locate at `AdditionalEndpointPrinter` not `printAppEndpoints` before my refactor. This commit just reuse `printAppEndpoints`, and does not change any code of it.

Do you think we could move this line to `printAppEndpoints`? it will make `./vela -n vela-system status addon-velaux --endpoint=true` to print this message.

Reply:
yes, I think we should add that.

```
$ ./vela -n vela-system status addon-velaux --endpoint=true
Please access the velaux from the following endpoints:
+---------+-----------+----------------------------+---------------------------+
| CLUSTER | COMPONENT |  REF(KIND/NAMESPACE/NAME)  |         ENDPOINT          |
+---------+-----------+----------------------------+---------------------------+
| local   | velaux    | Service/vela-system/velaux | http://192.168.65.4:30256 |
+---------+-----------+----------------------------+---------------------------+
```

It looks better.

Reply:
> yes, I think we should add that.
> 
> ```
> $ ./vela -n vela-system status addon-velaux --endpoint=true
> Please access the velaux from the following endpoints:
> +---------+-----------+----------------------------+---------------------------+
> | CLUSTER | COMPONENT |  REF(KIND/NAMESPACE/NAME)  |         ENDPOINT          |
> +---------+-----------+----------------------------+---------------------------+
> | local   | velaux    | Service/vela-system/velaux | http://192.168.65.4:30256 |
> +---------+-----------+----------------------------+---------------------------+
> ```
> 
> It looks better.

please note this line https://github.com/StevenLeiZhang/kubevela/blob/62c3e7ee4334b84d3c113e2b2ff784cacd1f88ce/references/cli/addon.go#L195

Reply:
I think this is fine, we are using `addon-velaux` in command:

```
vela -n vela-system status addon-velaux --endpoint=true
```

Reply:
> I think this is fine, we are using `addon-velaux` in command:
> 
> ```
> vela -n vela-system status addon-velaux --endpoint=true
> ```

It will always print "addon-velaux". It is ok for `vela -n vela-system status addon-velaux `, but may let user feel confused for `vela addon enable velaux`

Reply:
We can use this label to check and get the name `addons.oam.dev/name`